Broadcom/symantec-endpoint-protection/_meta/smart-descriptions.json

```
@@ -0,0 +1,52 @@
+[{
+    "value": "Allowed execution of {file.path} on {host.hostname} ({host.ip})",
+    "conditions": [
+        {
+            "field": "event.reason",
+            "value": "SONAR detection now allowed"
+        },
+        {
+            "field": "host.hostname"
+        },
+        {
+            "field": "host.ip"
+        },
+        {
+            "field": "file.path"
+        }
+    ],
+    "relationships": [
+        {
+            "source": "host.hostname",
+            "target": "file.path",
+            "type": "execute"
+        }
+    ]
+},{
+    "value": "Virus found on {host.hostname} ({host.ip}): {file.path} was {event.action}",
```

```suggestion
    "value": "Malware detected on {host.hostname} ({host.ip}): {file.path} was {event.action}",
```

Seems more appropriate